hazelcast/src/main/java/com/hazelcast/spi/impl/operationexecutor/slowoperationdetector/SlowOperationDetector.java

```diff
@@ -253,9 +266,8 @@ private void shutdown() {
             detectorThread.interrupt();
             try {
                 detectorThread.join(SLOW_OPERATION_THREAD_MAX_WAIT_TIME_TO_FINISH);
-            } catch (InterruptedException e) {
-                EmptyStatement.ignore(e);
-                // TODO: Interrupt flag is consumed here
+            } catch (InterruptedException ignored) {
+                detectorThread.interrupt();
```

Comment:
Good catch. I'll revert my change so we can do this in another PR (this one is already too big).